tests/migrations/test_commands.py

```diff
@@ -2555,6 +2555,57 @@ def test_squashmigrations_squashes(self):
             "  you can delete them.\n" % squashed_migration_file,
         )
 
+
+    def test_squashmigrations_squashes_already_squashed(self):
+        out = io.StringIO()
+
+        with self.temporary_migration_module(
+            module="migrations.test_migrations_squashed_complex"
+        ) as migrations_dir:
+            call_command(
+                "squashmigrations",
+                "migrations",
+                "3_squashed_5",
+                "--squashed-name",
+                "double_squash",
+                stdout=out,
+                interactive=False,
+            )
+
+            # this file rename allows for testing that the file loader is properly handling
+            # doubly-squashed migrations cleanly (this rename in practice seems to help with l)
+            old_squashed_migration_file = os.path.join(
+                migrations_dir, "0001_double_squash.py"
+            )
+            squashed_migration_file = os.path.join(
+                migrations_dir, "9999_double_squash.py"
+            )
+            os.rename(
+                old_squashed_migration_file,
+                squashed_migration_file,
+            )
+            # confirm the replaces mechanism holds the squashed migration (and not what it squashes, as the squash operations are what end up being used)
+
+            with open(squashed_migration_file, "r", encoding="utf-8") as fp:
+                content = fp.read()
+                # HACK Really I would just like to import the migration and do a Real Python List Comparison
```

Comment:
I was looking around and it looks like most of the migration tests are doing these sorts of string comparisons (along with the `WITH_BLACK` toggles to make the formatting-dependent expected cases)